distribution/lib/Standard/Base/0.0.0-dev/src/Any.enso

```diff
@@ -20,6 +20,26 @@ from project.Function import const
    be used in that position.
 @Builtin_Type
 type Any
+    ## GROUP Conversions
+       Generic conversion of an arbitrary Enso value to requested type
+
+       Arguments:
+       - typ: the requested type.
```

Comment:
Looking at the tests I can see that if the `from` conversion supported additional optional arguments, we can also pass these in the `to` conversion, right?

But I see no such information in the doc here that it may be possible to pass additional arguments. I'd appreciate this information and maybe an example - without them this feature is hard to discover.

Reply:
Documented in 0501581